### DISCUSSION TITLE
About  proxy  access logs

### DISCUSSION THREAD

**Discussion by eddielishan** [linkerd/linkerd2#8476] [Q&A]
https://github.com/linkerd/linkerd2/discussions/7847
“You can use an edge proxy for your control plane component that runs stable-2.11.x”

my linkerd  control plane version stable-2.11.2.  I set annotations  proxy version  edge-22.2.1  as follows

      **annotations:
        config.linkerd.io/access-log: json
        config.linkerd.io/proxy-version: edge-22.2.1**

bug access logs Not effective，If you do not upgrade the control plane version.Other adjustments are needed.thanks

**Comment by omidraha** [comment]
Hi,
To [enable](https://linkerd.io/2.14/features/access-logging/) "HTTP Access Logging",
Where should I apply this setting? `config.linkerd.io/access-log:json`
Based on this `emojivoto` [example](https://linkerd.io/2.14/getting-started/), I would like to enable it for it.
And how can I enable it for specific `POD` or all `pods` in one `namespace`?

**Reply by hawkw** [reply]
The annotation can be added to any Kubernetes resource you can annotate with `linkerd.io/inject: "enabled"`. For example, if you inject a Deployment or Pod, you can add the access log annotation to that Deployment or Pod. If you want to enable access logging for all injected pods in a namespace, you can instead add the `config.linkerd.io/access-log` annotation to that namespace.

Hope that helps!
